src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/migration/Cas2AssessmentMigrationJob.kt

```diff
@@ -0,0 +1,47 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.migration
+
+import org.slf4j.LoggerFactory
+import org.springframework.data.domain.Slice
+import org.springframework.transaction.support.TransactionTemplate
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.Cas2ApplicationEntity
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.Cas2ApplicationRepository
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.Cas2AssessmentEntity
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.Cas2AssessmentRepository
+import java.time.OffsetDateTime
+import java.util.UUID
+
+class Cas2AssessmentMigrationJob(
+  private val assessmentRepository: Cas2AssessmentRepository,
+  private val applicationRepository: Cas2ApplicationRepository,
+  private val transactionTemplate: TransactionTemplate,
+) : MigrationJob() {
+  private val log = LoggerFactory.getLogger(this::class.java)
+  override val shouldRunInTransaction = true
+
+  override fun process() {
+    log.info("Starting Cas2 Assessment Migration process...")
+
+    var page = 1
```

Comment:
How does `page` get used?

Reply:
ah yeah good spot, I don't think we need this and was copy/pasted from another different type of migration.
It did make me think that the logging could be more informative, so I've added `log.info("Saving assessment for ${application.id}")` before the transaction instead of the previous 'Getting page' log, sound good?